packages/hawtio/src/plugins/shared/attributes/AttributeTable.tsx

```diff
@@ -1,13 +1,11 @@
 import { PluginNodeSelectionContext } from '@hawtiosrc/plugins/context'
-import { JmxContentMBeans } from '@hawtiosrc/plugins/shared/JmxContentMBeans'
 import { AttributeValues } from '@hawtiosrc/plugins/shared/jolokia-service'
 import { humanizeLabels } from '@hawtiosrc/util/strings'
 import { Panel } from '@patternfly/react-core'
 import { Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table'
 import { JolokiaErrorResponse, JolokiaSuccessResponse } from 'jolokia.js'
 import React, { useContext, useEffect, useState } from 'react'
-import { HawtioEmptyCard } from '../HawtioEmptyCard'
-import { HawtioLoadingCard } from '../HawtioLoadingCard'
+import { HawtioEmptyCard, HawtioLoadingCard, JmxContentMBeans } from '@hawtiosrc/plugins/shared'
```

Comment:
they are within the same `shared` sub package, so I'd prefer to keep the relative path specifications. My rules are: use `@hawtiosrc` when cross-referencing, otherwise use relative.